packages/amesos2/src/Amesos2_Umfpack_TypeMap.hpp

```diff
@@ -117,14 +117,14 @@ struct TypeMap<Umfpack,double>
 template <>
 struct TypeMap<Umfpack,std::complex<float> > // provide conversion from std::complex<float> to std::complex<double>
 {
-  typedef std::complex<double> type;
+  typedef Kokkos::complex<double> type;
```

Comment:
Similar to the above, I think this need additional specializations for Kokkos::complex (which would include the Kokkos::complex typedef, rather than replacing the std::complex)